variables.tf

```diff
@@ -80,3 +80,9 @@ variable "prometheus_persistence_storage" {
   default     = false
   description = "Enable persistence storage for Prometheus"
 }
+
+variable "additional_scrape_configs" {
```

Comment:
Rename this `prometheus_additional_scrape_configs` and write documentation in usage.md on how to use this.